prelude/toolchains/conan/TARGETS

```diff
@@ -0,0 +1,57 @@
+native.export_file(
```

Comment:
This file should be named TARGETS.v2, since otherwise internal stuff gets confused

Reply:
Done.